src/Components/HomePage.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import TimerModal from './TimerModal';
+import { Link } from 'react-router-dom';
+
+const HomePage = () => {
+    const [modalState, setModalState] = useState({
+        isModalOpen: false
+    });
+
+    const openModal = () => setModalState({ ...modalState, isModalOpen: true });
+    const closeModal = () => setModalState({ ...modalState, isModalOpen: false });
```

Comment:
you can create 1 function which can do both, toggelModal and pass the parameter true/false, No need to create 2 functions @kaushaki-kumari